react/package.json

```diff
@@ -55,6 +55,8 @@
   "devDependencies": {
     "@acemarke/react-prod-sourcemaps": "^0.2.1",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@codecov/bundle-analyzer": "^1.6.0",
```

Comment:
You're safe to remove this package, as you're using a bundler you only need that package for BA to work.

Reply:
removed it, thanks!